modules/packages/ImageHelper/stb/README

```diff
@@ -0,0 +1,3 @@
+# README
+
+Bundled stb headers are pulled from https://github.com/nothings/stb
```

Comment:
This requires OSRB review -- is that done?

Reply:
Yes, @tzinsky is taking care of it